### PR TITLE
Enhance validate_api_difference_format 

### PR DESCRIPTION
**Pull request by LittleHeroZZZX**
增强 validate_api_difference_format 工具，新增如下检查项：
- 参数映射中的名称必须出现在参数签名中
- 参数映射表的参数顺序必须与签名中参数出现顺序一致
- Torch 中每个参数都要出现在参数映射表中

并据此修改差异文档

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/args_name_diff/torch.Tensor.diag.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ torch.Tensor.diag(diagonal=0)
 
 ### [paddle.Tensor.diag](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/diag_cn.html#diag)
 ```python
-paddle.Tensor.diag(offset=0, padding_value=0, name=None)
+paddle.Tensor.diag(x, offset=0, padding_value=0, name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/args_name_diff/torch.Tensor.diag.md]: 这个下面的x可以删掉

File: docs/guides/model_convert/convert_from_pytorch/api_difference/input_args_usage_diff/torch.nn.BatchNorm1d.md [hunk 2]
```diff
@@ -30,10 +30,8 @@ paddle.nn.BatchNorm1D(num_features,
...
 | eps           | epsilon      | 为了数值稳定加在分母上的值，仅参数名不一致。                                                                                                      |
 | momentum           | momentum      | 表示归一化函数中的超参数, PyTorch 和 Paddle 公式实现细节不一致，两者正好是相反的，需要转写。                                                                     |
-| -             | weight_attr  | 指定权重参数属性的对象。如果为 False, 则表示每个通道的伸缩固定为 1，不可改变。默认值为 None，表示使用默认的权重参数属性。                                                        |
-| -             | bias_attr    | 指定偏置参数属性的对象。如果为 False, 则表示每一个通道的偏移固定为 0，不可改变。默认值为 None，表示使用默认的偏置参数属性。                                                       |
 | -             | data_format  | 指定输入数据格式，PyTorch 无此参数，Paddle 保持默认即可。                                                                                        |
-| affine              | -                | 是否进行反射变换， Paddle 无此参数，需要转写。                                                                                                 |
+| affine              | weight_attr, bias_attr                 | 是否创建并使用可学习的仿射参数，Paddle 可以使用 weight_attr、bias_attr 分别控控制权重和偏置项的行为， PyTorch 只有一个开关，且默认值为 None，即不启用。       |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/input_args_usage_diff/torch.nn.BatchNorm1d.md]: 最后加一个：`需要转写`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/paddle_more_args/torch.nn.LazyBatchNorm1d.md [hunk 3]
```diff
@@ -24,10 +24,8 @@ paddle.nn.BatchNorm1D(num_features,
...
 | eps           | epsilon      | 为了数值稳定加在分母上的值，仅参数名不一致。                                                                                                      |
 | momentum      | momentum      | 表示归一化函数中的超参数, PyTorch 和 Paddle 公式实现细节不一致，两者正好是相反的，需要转写。                                                                     |
-| -             | weight_attr  | 指定权重参数属性的对象。如果为 False, 则表示每个通道的伸缩固定为 1，不可改变。默认值为 None，表示使用默认的权重参数属性。                                                        |
-| -             | bias_attr    | 指定偏置参数属性的对象。如果为 False, 则表示每一个通道的偏移固定为 0，不可改变。默认值为 None，表示使用默认的偏置参数属性。                                                       |
 | -             | data_format  | 指定输入数据格式，PyTorch 无此参数，Paddle 保持默认即可。                                                                                        |
-| affine        | -                | 是否进行反射变换， Paddle 无此参数，需要转写。                                                                                                 |
+| affine        | weight_attr, bias_attr                 | 是否进行仿射变换，Paddle 使用 weight_attr, bias_attr 两个参数分别控制权重和偏置控制是否进行放射变换，默认值为 False，PyTorch 使用一个参数统一控制，且默认值为 None，表示不启用。           |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/paddle_more_args/torch.nn.LazyBatchNorm1d.md]: 最后说明下：需要转写。

下同

**Review by zhwesky2010 — APPROVED** [review]
